app/Http/Controllers/AccountController.php

```diff
@@ -74,6 +79,12 @@ public function update(Request $request)
         $user->allow_profile_contact = $request->get('allow_profile_contact');
         $user->profile_intro = $request->get('profile_intro');
         $user->profile_slug = $request->get('profile_slug');
+
+        if ($request->file('profile_picture')) {
+            $image = $request->file('profile_picture');
+            Event::fire(new ProfilePictureUpdated($user, $image->getRealPath(), $image->getClientOriginalExtension()));
```

Comment:
Hm, good question. Since what we're really doing is saying "upload this picture", then it's more like a Job (and one which we should disable queuing for).